usaspending_api/api_docs/api_documentation/download/advanced_search_award_download.md

```diff
@@ -0,0 +1,58 @@
+## [Advanced Search Award Download](#Advanced_Search_Award_Download)
+**Route:** `/api/v2/download/awards/`
+
+**Method:** `POST`
+
+This route sends a request to the backend to begin generating a zipfile of award data in CSV form for download.
+
+### Request example
+
+```
+{
+    "filters": {},
```

Comment:
The request example gives the error `"At least one filter is required."`

Reply:
fixed